src/scaffold.ts

```diff
@@ -48,17 +51,19 @@ export function registerProjectGenerationCommands(): vscode.Disposable[] {
   ];
 }
 
-async function resourceScaffoldProjectRequest(item?: KafkaCluster | KafkaTopic) {
+async function resourceScaffoldProjectRequest(
```

Comment:
Any tests over this? Looks easily testable.

Reply:
https://github.com/confluentinc/vscode/issues/1592